src/transactions/feed/ClaimRewardFeedItem.tsx

```diff
@@ -0,0 +1,142 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import AppAnalytics from 'src/analytics/AppAnalytics'
+import { HomeEvents } from 'src/analytics/Events'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
+import { ClaimReward } from 'src/transactions/types'
+
+interface DescriptionProps {
+  transaction: ClaimReward
+}
+
+function Description({ transaction }: DescriptionProps) {
+  const { t } = useTranslation()
+  const txAppName = transaction.appName
+  const title = t('transactionFeed.claimRewardTitle')
+  const subtitle = t('transactionFeed.claimRewardSubtitle', {
+    context: !txAppName ? 'noTxAppName' : undefined,
+    txAppName,
+  })
+
+  return (
+    <View style={styles.contentContainer}>
+      <Text style={styles.title} testID={'ClaimRewardFeedItem/title'} numberOfLines={1}>
+        {title}
+      </Text>
+      <Text style={styles.subtitle} testID={'ClaimRewardFeedItem/subtitle'} numberOfLines={1}>
+        {subtitle}
+      </Text>
+    </View>
+  )
+}
+
+interface AmountDisplayProps {
+  transaction: ClaimReward
+  isLocal: boolean
+}
+
+function AmountDisplay({ transaction, isLocal }: AmountDisplayProps) {
+  const amountValue = new BigNumber(transaction.amount.value)
+  const tokenId = transaction.amount.tokenId
+
+  const textStyle = isLocal ? styles.amountSubtitle : [styles.amountTitle, { color: Colors.accent }]
+
+  return (
+    <TokenDisplay
+      amount={amountValue}
+      tokenId={tokenId}
+      showLocalAmount={isLocal}
```

Comment:
i think that this will show the local amount using the current exchange rate rather than the historical one from the transaction. i think we do this in a few places, we can go through and update later (but we could also add the `localAmount={transaction.amount.localAmount}` here)

Reply:
oh right, I just extracted this from earn, without giving much thought.